### PR TITLE
terraform code change S3

### PR DESCRIPTION
**Pull request by codvatechlabs**
JIRA Ticket ID -
nilesh@Nileshs-MacBook-Air s3-poc-terraform % terraform apply

Terraform used the selected providers to generate the following execution
plan. Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # aws_s3_bucket.s3bucket will be created
  + resource "aws_s3_bucket" "s3bucket" {
      + acceleration_status         = (known after apply)
      + acl                         = "private"
      + arn                         = (known after apply)
      + bucket                      = "my-tf-github-bucket"
      + bucket_domain_name          = (known after apply)
      + bucket_regional_domain_name = (known after apply)
      + force_destroy               = false
      + hosted_zone_id              = (known after apply)
      + id                          = (known after apply)
      + region                      = (known after apply)
      + request_payer               = (known after apply)
      + tags                        = {
          + "Environment" = "Dev"
          + "Name"        = "My bucket"
        }
      + tags_all                    = {
          + "Environment" = "Dev"
          + "Name"        = "My bucket"
        }
      + website_domain              = (known after apply)
      + website_endpoint            = (known after apply)

      + versioning {
          + enabled    = (known after apply)
          + mfa_delete = (known after apply)
        }
    }

Plan: 1 to add, 0 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

aws_s3_bucket.s3bucket: Creating...
aws_s3_bucket.s3bucket: Creation complete after 2s [id=my-tf-github-bucket]

Apply complete! Resources: 1 added, 0 changed, 0 destroyed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,11 @@
 # s3-poc-terraform
 This is to create s3 bucket using Terraform
+
+# steps to follow to run this project
+
```
Comment (sagar-dharaskar) [inline README.md:5]: Please provide documentation

File: backend.tf [hunk 2]
```diff
@@ -0,0 +1,8 @@
+terraform {
+  backend "s3" {
+    bucket = "ctl-terraform-state-ap-south-1"
+    key    = "dev/s3tfpoc"
```
Comment (sagar-dharaskar) [inline backend.tf:4]: Please provide proper key

File: main.tf [hunk 3]
```diff
@@ -0,0 +1,9 @@
+resource "aws_s3_bucket" "s3bucket" {
+  bucket = "my-tf-github-bucket"
+  acl    = "private"
+
+  tags = {
```
Comment (sagar-dharaskar) [inline main.tf:5]: Tag not valid

**Review by sagar-dharaskar — CHANGES_REQUESTED** [review]
Please work on all changes suggested

File: README.md (hunk 1 above)
Comment (codvatechlabs) [reply README.md:5]: I have updated code as per PR feedback

File: main.tf (hunk 3 above)
Comment (codvatechlabs) [reply main.tf:6]: I have updated code as per PR feedback

File: backend.tf (hunk 2 above)
Comment (codvatechlabs) [reply backend.tf]: I have updated code as per PR feedback

**Comment by codvatechlabs** [comment]
@sagar-dharaskar  : Please review and provide your approval

**Review by sagar-dharaskar — APPROVED** [review]

**Pull request merged**